pkg/cosign/git/gitlab/gitlab.go

```diff
@@ -70,6 +71,7 @@ func (g *Gl) PutSecret(ctx context.Context, ref string, pf cosign.PassFunc) erro
 		EnvironmentScope: gitlab.String("*"),
 	})
 	if err != nil {
+		ui.Warnf(ctx, "if you are using a self-hosted gitlab please set the \"GITLAB_HOST\" your server name.")
```

Comment:
nit: if in capital letter `"If you are using a self-hosted ...`

Reply:
done